client/app/bundles/course/assessment/submission/components/CheckboxFormGroup.jsx

```diff
@@ -26,7 +27,10 @@ export default class CheckboxFormGroup extends Component {
               return input.onChange(newValue);
             }}
             label={(
-              <div dangerouslySetInnerHTML={{ __html: option.option.trim() }} />
+              <div
+                style={option.correct && readOnly ? { backgroundColor: green50 } : null}
```

Comment:
Should we only show this to staff?

Reply:
Backend will not send the flag if user is not staff https://github.com/Coursemology/coursemology2/blob/master/app/views/course/assessment/question/multiple_responses/_multiple_response.json.jbuilder#L6

Reply:
what if it's not correct?

Reply:
Then the option won't be highlighted? Correct here means option should be chosen, not student's answer is correct.

Reply:
OK, I recall.